src/app.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const { server } = require('./server');
+
+server.listen(3000);
```

Comment:
Consider making the listening port number a constant or using a configuration file for improved flexibility and ease of maintenance 🛠️